Makefile

```diff
@@ -14,6 +14,8 @@
 # Set this to pass additional commandline flags to the go compiler, e.g. "make test EXTRAGOARGS=-v"
 EXTRAGOARGS:=
 DISABLE_ROOT_TESTS?=1
+FIRECRACKER_INIT_TIMEOUT_SECONDS?=
+FIRECRACKER_REQUEST_TIMEOUT_MILLISECONDS?=
```

Comment:
We shouldn't need these settings here, since environment variables evaluate to the empty string by default.